novelwriter/gui/doceditor.py

```diff
@@ -919,6 +934,10 @@ def keyPressEvent(self, event: QKeyEvent) -> None:
           * We also handle automatic scrolling here.
         """
         self._lastActive = time()
+
+        if self._handleVimKeyPress(event):
+            return
```

Comment:
Could we check `CONFIG.vimModeEnabled` here instead of inside the function so that we skip the function call if it isn't enabled? There is some overhead for calling functions and it's best to avoid it if it isn't necessary.

Something like:
```python
if CONFIG.vimModeEnabled and self._handleVimKeyPress(event):
    return
```

Reply:
Hmm, it seems the function always returns True if Vim Mode is enabled, so maybe the whole boolean return can be dropped and instead use:

```python
if CONFIG.vimModeEnabled:
    self._handleVimKeyPress(event)
    return
```